test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java

```diff
@@ -119,7 +122,7 @@ private void verifyPravegaPodRestart(Map<String, V1ContainerStatus> podStatusBef
     private V1Pod getTestPod(String className, String methodName, String podName) {
         log.info("Running test pod with security enabled :{}, transport enabled: {}", Utils.AUTH_ENABLED, Utils.TLS_AND_AUTH_ENABLED);
         V1Pod pod =  new V1PodBuilder()
-                .withNewMetadata().withName(podName).withNamespace(NAMESPACE).withLabels(ImmutableMap.of("POD_NAME", podName, "app", APP)).endMetadata()
+                .withNewMetadata().withName(podName).withNamespace(NAMESPACE).withLabels(ImmutableMap.of("POD_NAME", podName, "app", "pravega-system-tests")).endMetadata()
```

Comment:
Maybe you could leave the `APP` constant as it was, so we can avoid having this hardcoded string in the middle there.

Reply:
Yes, that was an unintended change.

Reply:
Ok, so why haven't you reverted it?